README.md

```diff
@@ -8,85 +8,95 @@
 
-In this getting started we will implement an instant search using Netlify, Algolia, and this plugin.  
-To start right away **you only need a Netlify site**.
+Here's how you can implement an instant search on your site using Netlify, Algolia, and this plugin. **All you need is an existing Netlify site.**
 
 ### Link your site to Algolia
 
-**Go to <https://crawler.algolia.com/admin/netlify>**
+First, go to the [Algolia Crawler for Netlify](https://crawler.algolia.com/admin/netlify) and click **Sign in to Algolia with Netlify**. We automatically create a new Algolia account if you don't have one.
 
-Click "Sign-in to Algolia with Netlify".
-We will automatically create a new Algolia account if you do not have one.
 <img src="/docs/screenshots/signin.png?raw=true" alt="Sign in to Algolia with Netlify">
 
-Authorize Algolia to access your Neltify account.
-The plugin will only update your plugin settings and add necessary environment variables, prefixed by `ALGOLIA_`.
+Then, authorize Algolia to access your Netlify account. The plugin only updates your plugin settings and adds necessary environment variables.
+
 <img src="/docs/screenshots/authorize.png?raw=true" alt="Authorize Algolia's OAuth">
 
-Search for your site name and click "Install".
-On this step we will automatically update your Netlify site to add a couple of environment variables required to use the plugin (prefixed by `ALGOLIA_`).
-We will also create an Algolia application with a dedicated [free plan](https://www.algolia.com/pricing/).
+In the [Crawler Admin Console](https://crawler.algolia.com/admin/netlify), search for your site, then click **Install**. We automatically update your Netlify site to add the environment variables (prefixed by `ALGOLIA_`) that you need to use the plugin.
+
+We also create an Algolia application with a dedicated [Free plan](https://www.algolia.com/pricing/).
 
 <img src="/docs/screenshots/search-site.png?raw=true" alt="Search for your site">
+
 <img src="/docs/screenshots/accept-install.png?raw=true" alt="Install in your site">
 
 The plugin is now correctly installed and ready to index your site.
+
 <img src="/docs/screenshots/installed.png?raw=true" alt="View your site information.">
 
 ### Indexing
 
-Now that you've installed our plugin, your next deploy to Netlify will launch a crawl that will populate an Algolia index.
-You can manually launch a new deploy by opening your latest deploy in Netlify and clicking on "Retry deploy" > "Deploy site".
+Once you've installed the plugin, your next Netlify deploy will trigger a crawl and populate an Algolia index. You can manually trigger a new deploy in Netlify by clicking **Retry deploy > Deploy site** on any deploy.
+
+Once we receive a build hook, the Algolia Crawler processes your website asynchronously. This operation takes some time, resulting in a short delay between the first deploy and the associated crawl. Your site and your Algolia index will be out of sync during that delay.
```

Comment:
That's 2 once in a row, is that fine?

Reply:
Hm no, I'll fix it.

Reply:
Handled in 04c6e90faac2675aa89e2176d2eec7d8.

Reply:
Handled in e5f9964c1b841df7f816587938e7124ca08ca5fb.